### PR TITLE
DAPI-1240 - add data quality insights migration specificities

### PR DESCRIPTION
**Pull request by AymericPlanche**
**Description**

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: migrate_pim/upgrade/upgrade_from_40_to_50.rst [hunk 1]
```diff
@@ -29,6 +29,52 @@ You have to make sure your system components are updated to the version required
 
     So there's no need to export and reimport data for this system.
 
+
+Updated System dependencies
+---------------------------
+
+Community Edition
+^^^^^^^^^^^^^^^^^
+TODO?
```
Comment (rfauglas) [inline migrate_pim/upgrade/upgrade_from_40_to_50.rst:38]: Who might know about this?

File: migrate_pim/upgrade/upgrade_from_40_to_50.rst [hunk 2]
```diff
@@ -29,6 +29,52 @@ You have to make sure your system components are updated to the version required
...
+
+Enterprise Edition
+^^^^^^^^^^^^^^^^^^
+The following libraries must be added to your operating system:
+
+.. code:: bash
+
+    $ apt-get install aspell-it aspell-sv aspell-da aspell-nl aspell-no aspell-pt-br
+
+Updated crontab definition
```
Comment (rfauglas) [inline migrate_pim/upgrade/upgrade_from_40_to_50.rst:48]: Shouldn't be at the end before "Migrating your custom code"?

File: migrate_pim/upgrade/upgrade_from_40_to_50.rst (hunk 1 above)
Comment (AymericPlanche) [reply migrate_pim/upgrade/upgrade_from_40_to_50.rst:38]: If no one logged it all along the year, I would say a git diff off the dockerfile between 4.x and master could be a good start.

File: migrate_pim/upgrade/upgrade_from_40_to_50.rst (hunk 2 above)
Comment (AymericPlanche) [reply migrate_pim/upgrade/upgrade_from_40_to_50.rst:48]: you're right, the crontab definition must be done at the very end, when the code is up-to-date and the migrations done.

File: migrate_pim/upgrade/upgrade_from_40_to_50.rst [hunk 3]
```diff
@@ -29,6 +29,52 @@ You have to make sure your system components are updated to the version required
...
+--------------------------
+
+Community Edition
+^^^^^^^^^^^^^^^^^
+
+Added
```
Comment (rfauglas) [inline migrate_pim/upgrade/upgrade_from_40_to_50.rst:54]: Indeed crontab change is new in the upgrade documentation... I wonder if it makes sense...
Furthermore you missed the "cron wrapper", 
pim:data-quality-insights:prepare-evaluations are duplicate commands in community and enterprise

File: migrate_pim/upgrade/upgrade_from_40_to_50.rst (hunk 1 above)
Comment (rfauglas) [reply migrate_pim/upgrade/upgrade_from_40_to_50.rst:38]: I finally understood what  you meant...    
Main diff in Dockerfile.base is:
apt-get --yes install apt-transport-https ca-certificates curl wget &&\
    wget -O /etc/apt/trusted.gpg.d/php.gpg https://packages.sury.org/php/apt.gpg &&\

File: migrate_pim/upgrade/upgrade_from_40_to_50.rst (hunk 3 above)
Comment (AymericPlanche) [reply migrate_pim/upgrade/upgrade_from_40_to_50.rst:54]: I don't know what you mean by "cron wrapper" - nothing related to Data quality insights feature.

regarding the `pim:data-quality-insights:prepare-evaluations` yes it as to be added to the crontab if you're migrating both CE or EE version, so it as to be documented that way.

Comment (AymericPlanche) [reply migrate_pim/upgrade/upgrade_from_40_to_50.rst:54]: and for me it make sense, if the fact that their are new cron to configure from a version to another is not in a migration guide so you've to tell the customer/integrator: play the 7 differences game between you're installed PIM and the "Install a pim from scratch" documentation to find what is new, what is deleted, what is updated ;)

We can discuss about that IRL ;)

**Pull request merged**